js/google_ima.js

```diff
@@ -272,6 +274,11 @@ require("../html5-common/js/utils/utils.js");
       this.registerUi = function()
       {
         this.uiRegistered = true;
+        if (_amc.ui.useSingleVideoElement && !this.sharedVideoElement && _amc.ui.ooyalaVideoElement[0] &&
+            (_amc.ui.ooyalaVideoElement[0].className === "video")) {
```

Comment:
Where is this className defined? It would be better if it can be fetched or received from somewhere. We can avoid issues that arise if this class name needs to be changed in the future.